pkgs/os-specific/linux/rtl8814au/default.nix

```diff
@@ -1,14 +1,14 @@
 { lib, stdenv, fetchFromGitHub, kernel }:
 
-stdenv.mkDerivation {
-  pname = "rtl8814au";
-  version = "${kernel.version}-unstable-2021-05-18";
+stdenv.mkDerivation rec {
+  name = "rtl8814au-${kernel.version}-${version}";
```

Comment:
why change it to name? this should be done automatically if you leave it as pname